bci_tester/data.py

```diff
@@ -258,8 +258,18 @@ def create_BCI(
 NODEJS_18_CONTAINER = create_BCI(build_tag="bci/nodejs:18")
 
 PYTHON36_CONTAINER = create_BCI(build_tag="bci/python:3.6")
```

Comment:
shouldn't the `available_versions` also specified for 3.6?

Reply:
I think the 3.6 container is available for any SP, so omitting it should be fine.